sys/ps/ps.c

```diff
@@ -130,7 +133,8 @@ void ps(void)
 #endif
                    sname, queued, p->priority
 #ifdef DEVELHELP
-                   , p->stack_size, stacksz, (void *)p->stack_start, (void *)p->sp
+                   , p->stack_size, stacksz, stack_free
+                   , (void *)p->stack_start, (void *)p->sp
```

Comment:
```suggestion
                   , p->stack_size, stacksz, stack_free,
                   (void *)p->stack_start, (void *)p->sp
```

Reply:
According to RIOTs coding convention, commas should always be after the parameter and not at the beginning of the line.

In the first line this is not possible due to the conditional compilation using the cpp: If `DEVELHELP` is not defined, `p->priority` would be already the last parameter. So placing the comma after `p->priority` (which would be what the coding convention says) would not compile without `DEVELHELP` defined. Thus, the comma is moved in front of the next parameter to also be only conditionally compiled. This exception does however not apply to the second line, as there the comma after `stack_free` does not need to be moved to the next line.